node/runtime/pangoro/src/pallets/bridge_messages.rs

```diff
@@ -17,8 +17,6 @@ frame_support::parameter_types! {
 		bp_pangolin::MAX_UNCONFIRMED_MESSAGES_IN_CONFIRMATION_TX;
 	pub const MaxUnrewardedRelayerEntriesAtInboundLane: MessageNonce =
 		bp_pangolin::MAX_UNREWARDED_RELAYERS_IN_CONFIRMATION_TX;
-	// 0x726f6f7400000000000000000000000000000000, b"root"
-	pub RootAccountForPayments: Option<AccountId> = Some(ConcatConverter::<_>::derive_substrate_address(&H160([0x72, 0x6f, 0x6f, 0x74, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0])));
```

Comment:
@xiaoch05 @HackFisher Please confirm that we can delete this safely.

Reply:
This is still being used, please do not delete it.

Reply:
how to pay the s2s relay fee for SourceRoot accont in new version?
This is used by remote governance

Reply:
> how to pay the s2s relay fee for SourceRoot accont in new version? This is used by remote governance

Need to check from the upstream code. They deleted this.

Reply:
I will raise a PR to resolve this.

Reply:
https://github.com/darwinia-network/darwinia-common/pull/1452